src/Vasont.Inspire.Models.csproj

```diff
@@ -19,15 +19,16 @@
     <RepositoryUrl>https://github.com/vasont-systems/Vasont.Inspire.Models</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <PackageTags>CCMS</PackageTags>
-    <PackageReleaseNotes>Modified Editor save model to support standardized Contents property and EditorType. These will assist Inspire application in understanding what to do with the contents from an editor.</PackageReleaseNotes>
-    <Version>1.1.15</Version>
+    <PackageReleaseNotes>updated packages</PackageReleaseNotes>
```

Comment:
Needs more detail. I would suggest describing model changes a little bit more in detail. This goes into the Nuget package.